src/averageAges.js

```diff
@@ -1,61 +1,53 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function calculateAge(people) {
+  const ageArr = people.map((person) => person.died - person.born);
+  const sumAge = ageArr.reduce((a, b) => a + b);
+
+  return sumAge / ageArr.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArr = people.filter((person) => person.sex === 'm');
+  const menCenturyArr = menArr
```

Comment:
you create this variable even if there's no century passed. Try to assign the value to the single variable, but make it conditional.
Ternary will help you perfectly with this

And same issue with naming as I described in my previous comment